HACKING.md

```diff
@@ -79,6 +79,10 @@ source .tox/unit/bin/activate
 pytest -v test/test_real_pebble.py
 ```
 
+To test Pebble functionality that has been merged but not yet released, you can run
+`go install github.com/canonical/pebble/cmd/pebble@master`. This otherwise works in
```

Comment:
Now that CI is actually using `@master`, shall we just remove this paragraph and update the paragraph further above to say to use `@master`?